src/main/resources/db/migration/all/20220803121541__characteristic_table.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE characteristic (
+    id UUID NOT NULL,
+    name TEXT NOT NULL,
+    service_scope TEXT DEFAULT '*',
```

Comment:
these should perhaps be `NOT NULL` too since I'm assuming `*` means apply to all services / models?